cache/cache.go

```diff
@@ -101,13 +101,17 @@ func (w *cachedWriter) Write(data []byte) (int, error) {
 }
 
 // Cache Middleware
-func Cache(store *CacheStore) gin.HandlerFunc {
+func Cache(store CacheStore) gin.HandlerFunc {
```

Comment:
@javierprovecho this is blocking bug, because of this I can't call this function with a store object (RedisStore, and others)